{{ cookiecutter.project_shortname }}/run-tests.sh

```diff
@@ -2,24 +2,24 @@
 {% include 'misc/header.py' %}
 
 # Usage:
-#   env DB=postgresql12 ES=es7 CACHE=redis MQ=rabbitmq ./run-tests.sh
+#   env DB=postgresql12 SEARCH=elasticsearch7 CACHE=redis MQ=rabbitmq ./run-tests.sh
 
 # Quit on errors
 set -o errexit
 
 # Quit on unbound symbols
- set -o nounset
+set -o nounset
 
 # Always bring down docker services
-function cleanup {
-    docker-services-cli down
+cleanup() {
+    eval "$(docker-services-cli down --env)"
 }
 trap cleanup EXIT
 
-python -m check_manifest --ignore ".travis-*"
+python -m check_manifest --ignore ".*-requirements.txt"
 python -m sphinx.cmd.build -qnNW docs docs/_build/html
 # TODO: Remove services below that are not neeed (fix also the usage note).
-docker-services-cli up ${DB} ${ES:-es} ${CACHE:-redis} ${MQ:-rabbitmq}
+eval "$(docker-services-cli up ${DB} ${SEARCH:-elasticsearch} ${CACHE:-redis} ${MQ:-rabbitmq} --env)"
```

Comment:
Just a reminder that this line must be adapted once [`docker-services-cli` changes](https://github.com/inveniosoftware/docker-services-cli/pull/19) are applied.

Reply:
Question: Should we have a default for DB? All the other services have a default.

Reply:
It would be useful. The problem is that sqlite is used as default, and shouldn't start any service...... it would be useful if you could do:

```
eval "$(docker-services-cli up ${DB:-sqlite} --env)"
```

Then you can basically do both:

```
env DB=postgresql ./run-tests.sh
```

or 

```
./run-tests.sh
```
(without a default this late one won't work)